app/bundles/LeadBundle/Entity/LeadListRepository.php

```diff
@@ -859,8 +859,15 @@ public function getListFilterExpr($filters, &$parameters, QueryBuilder $q, $not
 
             switch ($details['field']) {
                 case 'hit_url':
-                    $operand = (($func == 'eq') || ($func == 'like')) ? 'EXISTS' : 'NOT EXISTS';
-
+                case 'referer':
+                case 'source':
+                case 'url_title':
+                    $operand = in_array($func, ['eq', 'like', 'regexp' ,'notRegexp']) ? 'EXISTS' : 'NOT EXISTS';
+
+                    $column = $details['field'];
+                    if($column == 'hit_url'){
```

Comment:
There are several places where the CS is not met. Could you run this PR through the pre-commit hook which will fix the CS?